frontend/plugins/frontline_ui/src/widgets/automations/modules/facebook/components/bots/components/FacebookBotPageSelectorSteps.tsx

```
@@ -0,0 +1,107 @@
+  const selectedAccountId =
+    useAtomValue(selectedFacebookAccountAtom) || accountId;
+  const selectedPageId = useAtomValue(selectedFacebookPageAtom);
+  const [isOpenAccountSheet, setOpenAccountSheet] = useAtom(
+    isOpenFacebookBotSecondarySheet,
+  );
+
+  const isLastStep = activeStep === 2 && selectedAccountId && selectedPageId;
+
+  const onClick = () => {
+    if (isLastStep) {
+      isOpenAccountSheet && setOpenAccountSheet(false);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)